rpcs3/Emu/RSX/Capture/rsx_replay.cpp

```diff
@@ -103,6 +103,9 @@ namespace rsx
 		// apply memory needed for command
 		for (const auto& state : replay_cmd.memory_state)
 		{
+			if (is_stopped())
```

Comment:
You shouldn't need to do this outside the main control loop. Otherwise every function in the emulator would have a is_stopped() check. It's a bit much.